dist/index.js

```diff
@@ -2900,11 +2900,11 @@ function run() {
                 core.error(`jdksource should be one of [upstream, github-hosted, install-jdk]. Found: ${jdksource}`);
             }
             if (buildList !== 'openjdk' &&
```

Comment:
Had you also wanted to change this to use the same pattern as ones below !buildList.startsWith('openjdk'), or intentionally leaving as is?  (same question for change in aqa.ts)

I see now, I am guessing its because currently openjdk has no sub-directories.